docs/gallery/static/impulse_response_measurement.ipynb

```diff
@@ -0,0 +1,500 @@
x9+7dUiopUaTLtTt16hTGjh2Lzp07o3Pnzpg8eXKR15oYjq7vO6nLly/D09MTP/74o4FLSFTR9bplZGRg5cqV6NmzJ7y8vDBo0CD6zCwjul67w4cPY9CgQWjbti369OmDNWvWIC8vr5RKSwDgyZMnmDVrFnx8fODp6YmbN28WeUxAQABGjBgBLy8vfPnllzh37pzO56VgUcGVK1ewbt06jBs3DgcPHkTdunUxbdo0JCcnq0z//PlzzJ07FwMGDMChQ4fQqVMn/PTTT3j37l0pl5zoeu0eP36Mnj17Ytu2bdi7dy+qVq2KqVOnIj4+vpRLTnS9dlKxsbHYsGEDWrRoUUolJbJ0vW4CgQBTpkxBbGwsVq5ciRMnTmDu3LmoUqVKKZec6HrtLl26hM2bN2P8+PE4duwY/vjjD1y9ehV//fVXKZf885aTkwN3d3f8+uuvWqWPiYnBzJkz4enpicOHD2PYsGFYsmQJHjx4oNuJGSLn22+/ZVasWME+FolEjI+PD7N3716V6X/77TdmxowZctu+++47ZunSpQYsJVFF12unSCgUMh06dGDOnTtnoBISdYpz7YRCITN69Gjm1KlTzPz585nZs2eXQkmJLF2v27Fjx5j+/fszAoGglEpI1NH12q1YsYKZOHGi3La1a9cyY8aMMWQxiQYeHh7MjRs3NKbZsGED8/XXX8tt++2335ipU6fqdC6qWZQhEAgQFBSE1q1bs9u4XC5atWqFFy9eqDzmxYsXaNWqldw2Ly8vBAYGGrSsRF5xrp2i3NxcCIVCVK5c2VDFJCoU99rt2rULtra2+PLLL0uhlERRca7b7du30bRpU6xcuRI9evTAkCFDsGfPHohEotIqNkHxrl2zZs3w5s0btqk6Ojoa9+7dQ9u2bUulzKR4AgMD5a4zIIlRtP1elOLrs1AVXWpqKkQiEWxtbeW229raIjw8XOUxSUlJKtMnJSUZqphEheJcO0WbNm2Cvb29UvBPDKs41+7Zs2c4c+YMDh8+XAolJKoU57rFxMQgICAAPj4+2LBhA6KiorBy5UoIhUKMHz++FEpNgOJdOx8fH6SmpmLs2LFgGAYikQiDBw/GmDFjSqHEpLjUxShZWVnIzc2FqampVvlQsEgIgH379uHKlSvYvn07TExMyro4RIOsrCzMmzcPc+fOhbW1dVkXh+iAYRjY2Nhg7ty54PF4aNCgAeLj43HgwAEKFsu5gIAA7N27F7/99hsaN26MqKgorF69Grt27cLYsWPLunjEwChYlGFtbQ0ej6fUwTc5ORl2dnYqj7Gzs9MpPTGM4lw7qQMHDmDfvn3YsmUL3N3dDVlMooKu1y46OhqxsbGYPXs2u00sFgMAWrdujRMnTsDV1dWwhSbFes/Z29uDz+eDx+Ox22rVqoWkpCQIBAIYGRkZtMxEojjXbtu2bejduzfb7cPNzQ05OTlYunQpxowZAy6XerWVR+piFAsLC61rFQEaDS3HyMgI9evXx8OHD9ltYrEYjx49QtOmTVUe07RpUzx69Ehum7+/P5o0aWLQshJ5xbl2ALB//37s2rULmzZtQsOGDUujqESBrteuZs2aOHLkCA4dOsT+69ChAzw9PXHo0CFUrVq1NIv/2SrOe65Zs2aIiopig3sAiIyMhL29PQWKpag41y43NxccDkdumzRAZBjGcIUlJdKkSRO56wxIYhRN34uqULCoYMSIETh9+jTOnz+P9+/fY/ny5cjJyUG/fv0AAPPmzcPmzZvZ9N988w3u37+PgwcPIjw8HNu3b8fr168xZMiQsnoKny1dr92+ffuwbds2zJs3D05OTkhMTERiYiKys7PL6il8tnS5diYmJnBzc5P7Z2VlBXNzc7i5uVHQUYp0fc8NHjwY6enpWL16NSIiInD37l3s3bsXX3/9dVk9hc+Wrteuffv2OHHiBC5fvoyYmBj4+flh27Zt6NChg1xNMTGs7OxsBAcHIzg4GICkH3BwcDDi4uIAAJs3b8a8efPY9IMHD0ZMTAw2bNiA8PBwHDt2DL6+vhg+fLhO56VmaAU9evRASkoKtm3bhqSkJNStWxebNm1iq+bj4uLkqtubNWuGpUuXYsuWLfjrr79QrVo1rF69Gm5ubmX1FD5bul67EydOQCAQKM1XNW7cOEyYMKFUy/650/XakfJB1+vm6OiITZs2Ye3atRg2bBgcHBzwzTff4Lvvviurp/DZ0vXa/fDDD+BwONi6dSsSEhJgbW2NDh06YPLkyWX1FD5Lr1+/xsSJE9nH69atAwD07dsXCxYsQGJiIhs4AoCLiwvWr1+PtWvX4siRI6hSpQp+//13eHl56XReDkP1x4QQQgghRA36qU4IIYQQQtSiYJEQQgghhKhFwSIhhBBCCFGLgkVCCCGEEKIWBYuEEEIIIUQtChYJIYQQQohaFCwSQgghhBC1KFgkhBBCCCFqUbBICCFaWrBgAX788cdSP++5c+fQqVOnUj8vIYQAtIILIYQAADw9PTXuHzduHEaMGAGGYWBlZVVKpZLIzc1FdnY2bG1tS/W8hBACULBICCEAgMTERPbvq1evYtu2bThx4gS7zdzcHObm5mVRNEIIKVP8si4AIYSUB/b29uzflpaW4HA4ctsASTN0RkYG1qxZAwAYP3483NzcwOPxcP78eRgZGWHSpEnw8fHBn3/+iWvXrsHW1hY///wz2rZty+bz7t07bNy4EU+fPoWZmRlat26NH3/8EdbW1irLdu7cOaxZswY3b94EAGzfvh23bt3CiBEjsG3bNqSnp8Pb2xu///47LCwsVObx4cMH/Pnnn3j27BkEAgGcnZ0xffp0tGvXrgSvGiHkc0B9FgkhpAQuXLiAypUrY//+/RgyZAhWrFiBX3/9FU2bNsXBgwfRpk0bzJs3D7m5uQCAjIwMTJo0CfXq1cOBAwewceNGJCcn47ffftPpvNHR0bh58ybWrVuH9evX48mTJ9i3b5/a9CtXrkR+fj527tyJI0eOYNq0aVRTSgj
```

Comment:
Great section for teaching purposes, but I'm not sure if it's the right place in the pyfar application gallery. Any opinions?

Reply:
I thought with such a real-life example it would be a good idea to point out, that this is only a basic example and that more should be done, depending on what you want to do. I'd like to keep it here.

Reply:
Ok, I'm fine with that :)